lib/bambora/www_form_client.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Bambora
+  # The base class for making XML requests.
+  class WWWFormClient < Bambora::RestClient
+    CONTENT_TYPE = 'application/x-www-form-urlencoded'
+
+    ##
+    # Make a POST Request.
+    #
+    # @param path [String] Indicating request path.
+    # @param body [Hash] Data to be sent in the query parameters of the request.
+    # @param api_key [String] Indicating the API Key to be used with the request.
+    #
+    # @return [Hash] Indicating success or failure of the operation.
+    def post(path:, body:, api_key:)
+      Bambora::ResponseAdapterFactory.for(
+        super(
+          path: "#{path}?#{WWWFormParameters.new(body: body)}",
```

Comment:
Excon's and Faraday's docs show that you can pass query parameters into the `body` and they get formatted into the URL, but this wasn't working for me, so I'm formatting them here and tacking them onto the end of the URL.

Reply:
Is it worth adding this comment in the code as to _why_ you're building the query parameter string?

Reply:
Yes! I'd like to get this PR merged though, so I'll add that in a follow up.